doc/UserGuide.md

```diff
@@ -48,6 +48,9 @@ Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/ADDRESS [t/TAG]...`
 Examples: 
 * `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
 * `add Betsy Crowe pp/1234567 e/betsycrowe@gmail.com pa/Newgate Prison t/criminal t/friend`
+### Sort all persons : `sort`
+Sorting all person from the address book in alphabetical ordering.
+Format: `sorting`
```

Comment:
"sorting" command name is not consistent with all the other command names like "add" "list" "find". It is btter to just use "sort" as command word.